hkmc2/shared/src/main/scala/hkmc2/syntax/Parser.scala

```diff
@@ -273,26 +281,27 @@ abstract class Parser(
             yeetSpaces match
             case (tok @ BRACKETS(Indent | Curly, toks), loc) :: _ /* if subRule.blkAlt.isEmpty */ =>
               consume
+              if qualifiers.nonEmpty then
+                err((msg"Blocks cannot be annotated" -> S(loc) :: Nil))
```

Comment:
This error should be diff-tested.

Reply:
I added a test, but this case is tricky because the condition under which it appears is: the foremost token is an identifier; this identifier is also keyword; this keyword has a rule starting with it in the `prefixRules1, but there is no rule starting with this keyword in the current `rule`.

Reply:
I tried adding `???` to understand where it's used by seeing where it crashes, but I found that all of these tests are marked as todo or fixme.

Reply:
That's a great idea! I overlooked the fact that it called `blockOf`.